Core/GDCore/Project/ObjectGroup.h

```diff
@@ -26,7 +29,36 @@ namespace gd {
 class GD_CORE_API ObjectGroup {
  public:
   ObjectGroup(){};
-  virtual ~ObjectGroup(){};
+

```

Comment:
Almost sure you can revert all the changes in this file, you don't need this for ObjectGroup.